lab_1_keywords_tfidf/main.py

```diff
@@ -156,4 +221,13 @@ def extract_significant_words(chi_values: dict[str, float], alpha: float) -> Opt
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if (dict_type_check(chi_values, str, float) and chi_values
+        and alpha in [0.05, 0.01, 0.001]):
+
+        criterion = {0.05: 3.842, 0.01: 6.635, 0.001: 10.828}
+        significant_chi_words = {}
```

Comment:
could you compose this dictionary using dict comprehension?

Reply:
done